buildAndReleaseTask/installers/pulumi.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import * as tl from "azure-pipelines-task-lib/task";
+import * as lib from "azure-pipelines-tool-lib";
+import * as path from "path";
+
+export async function installPulumiWithToolLib(expectedVersion: string, latestPulumiVersion: string) {
+
+  let downloadUrl: string;
+  const os: string = tl.osType();
+  tl.debug(tl.loc("OSDETECTED", os));
+  if (expectedVersion.toLowerCase() === "latest") {
```

Comment:
I think it would also be worth to call out that, we won't check if the `expectedVersion` > `latestPulumiVersion`, which would result in an error later on in this function. I think it is fine that we don't guard against non-existent versions, since we would then want the user to be explicit about which version they want this task to install, instead of implicitly using the latest version because the version they request didn't exist.

Reply:
done